### ISSUE TITLE
Adding a "Lists" tab with swipe functionality

### ISSUE THREAD

**Issue by nrgeen** [tuskyapp/Tusky#4053]
Currently I am missing a direct way to access my lists in Tusky. I propose to add a new tab called "Lists" that allows to view and navigate between all available lists.

Functionality:
1. a new tab "Lists" can be added to the bottom navigation bar.
2. clicking on the "Lists" tab will open the "first" list by default.
3. you can switch between the lists by swiping to the left or right.
4. the order of the lists can be adjusted in the settings.

Thank you for your attention and great work on Tusky! I look forward to implementing this feature. 😁

Best regards,
Nrgeen

**Comment by Tak** [comment]
Hi, did you see that in `Account Preferences / Tabs`, you can add custom tabs for your lists?

**Comment by nrgeen** [comment]
@Tak Thank you for the tip. I am aware of the function.

I have the following situation:
Currently I have used up the possible 5 spaces for tabs. I cannot add any more.
I maintain several lists in addition to the startpage. The posts there do not appear on my startpage so that I can get some order into the timeline. To get to all the lists, I currently have to open the sidebar, then click on Lists and then select each list individually, which is very click-intensive.

**Comment by Lakoja** [comment]
>     1. a new tab "Lists" can be added to the bottom navigation bar.
> 
>     2. clicking on the "Lists" tab will open the "first" list by default.
> 
>     3. you can switch between the lists by swiping to the left or right.

I think, I generally like the idea.

I (only) see two problems:

* How to distinguish this from the existing "one list as tab" functionality? (one might ask "Why do have two ways for the same functionality?")
* Swiping will conflict with the swiping between tabs, but maybe a dropdown would work?

**Comment by connyduck** [comment]
We could just display the list of lists in a tab? Clicking one would open the list in a new activity. This would be relatively easy to implement and at least it saves you opening the menu and selecting "Lists".

**Comment by nrgeen** [comment]
So far I have only been a Tusky user and am not familiar with the design choices. Am I right in assuming that you have decided to leave a maximum of 5 icons in the table bar? Otherwise, why not just allow more icons? A recommendation could also be added that the bar becomes more confusing with more than 5 icons and may even start to scroll. 

@connyduck Allowing the list overview in the tabs would be a start and would at already save one click.

Another possibility would be for the list menu to open a kind of pop-up. In concrete terms, it could look like a kind of window or floating layer that opens above the other control elements. Both the list and a swipe between the lists would then be possible.

Thank you for your appreciation of this request!

**Comment by connyduck** [comment]
Support for infinite tabs was just merged and will be in the next release.

**Comment by nrgeen** [comment]
That's interesting!

Does this mean that the limit of 5 tabs will also be removed?

Will a new tab limit be introduced?

**Comment by connyduck** [comment]
With "infinite tabs" I meant that the limit has been removed.

**Comment by nrgeen** [comment]
That's great!
Doesn't that solve my problem? Then I could add several lists as tabs and swipe through the different timelines. Or do I understand this wrong?

An additional tab that then displays all the lists would still be a useful feature.

Is there already a rough timetable for the release or even beta versions?

**Comment by connyduck** [comment]
You can already test the feature with Tusky nightly: https://github.com/tuskyapp/faq/blob/main/README.md#where-can-i-get-tusky

**Comment by nrgeen** [comment]
I've had the nightly version running since yesterday and it's doing a good job. Thanks for the tip. I will now take this as an opportunity to close the issue.
Nevertheless, a question. First I tried to install the nightly via FDroid. The version I got from there does not have the "infinite tabs" function, so I had to install the version from the PlayStore first. According to the description at https://github.com/tuskyapp/faq/blob/main/README.md#where-can-i-get-tusky, this should be the same app version, or am I mistaken?
